### ISSUE TITLE
YAML inputs should only consider key values

### ISSUE THREAD

**Issue by himynamesdave** [signalscorps/file2stix#54]
only key values are considered. e.g. `KEY: VALUE`, only `VALUE` would remain.

However both key and value are currently being captured

 file2stix --input-file tests/file_inputs/yaml/input.yml

**Comment by himynamesdave** [comment]
@shanthanu9 yaml. sorry. title fixed

**Comment by himynamesdave** [comment]
@shanthanu9 for the input file2stix --input-file tests/file_inputs/yaml/input.yml

AS observables are still being extracted

**Comment by himynamesdave** [comment]
@shanthanu9 even simpler, all items in a yaml/yml file should be consider

lists, keys, values, comments, etc.

**Comment by himynamesdave** [comment]
@shanthanu9 nope, not quite. try

file2stix --input-file tests/file_inputs/yaml/input.yaml

See there are ip addresses in the value fields.

but bundle does not contain them

```json
{
    "type": "bundle",
    "id": "bundle--f59c1748-ae2c-4683-ab2b-4604eb08a877",
    "objects": [
        {
            "type": "identity",
            "spec_version": "2.1",
            "id": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-01-01T00:00:00.000Z",
            "modified": "2022-01-01T00:00:00.000Z",
            "name": "file2stix",
            "description": "https://github.com/signalscorps/file2stix/",
            "identity_class": "organization",
            "sectors": [
                "technology"
            ]
        },
        {
            "type": "marking-definition",
            "spec_version": "2.1",
            "id": "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9",
            "created": "2017-01-20T00:00:00.000Z",
            "definition_type": "tlp",
            "name": "TLP:WHITE",
            "definition": {
                "tlp": "white"
            }
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--6c0df49d-02c8-40be-aafe-fc0d1ab67eee",
            "created": "2022-09-21T15:18:09.856724Z",
            "modified": "2022-09-21T15:18:09.856724Z",
            "name": "AS15139",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ autonomous-system:number = '15139' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-21T15:18:09.856724Z",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--5f1453be-7a5c-489a-8a5d-b3cb237a67ba",
            "created": "2022-09-21T15:18:09.86787Z",
            "modified": "2022-09-21T15:18:09.86787Z",
            "name": "AS15140",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ autonomous-system:number = '15140' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-21T15:18:09.86787Z",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--ecce5385-4873-4d35-a8d5-50224341327c",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-21T15:18:09.853556Z",
            "modified": "2022-09-21T15:18:09.853556Z",
            "relationship_type": "default-extract",
            "source_ref": "report--51e1f5a8-f6a3-4849-a582-b2af9af0a51f",
            "target_ref": "indicator--6c0df49d-02c8-40be-aafe-fc0d1ab67eee",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--3d98bd94-546e-4cc2-8e25-8e40daaa3c57",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-21T15:18:09.853556Z",
            "modified": "2022-09-21T15:18:09.853556Z",
            "relationship_type": "default-extract",
            "source_ref": "report--51e1f5a8-f6a3-4849-a582-b2af9af0a51f",
            "target_ref": "indicator--5f1453be-7a5c-489a-8a5d-b3cb237a67ba",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "extension-definition",
            "spec_version": "2.1",
            "id": "extension-definition--c8ea5ecb-f4a3-45e7-94de-9b9ba05161af",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-01-01T00:00:00.000Z",
            "modified": "2022-01-01T00:00:00.000Z",
            "name": "MISP Warning Lists",
            "description": "This schema adds MISP Warning List matches to extracted Objects",
            "schema": "https://github.com/MISP/misp-warninglists",
            "version": "1.0",
            "extension_types": [
                "property-extension"
            ]
        },
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--51e1f5a8-f6a3-4849-a582-b2af9af0a51f",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-21T15:18:09.853556Z",
            "modified": "2022-09-21T15:18:09.853556Z",
            "name": "File converted: input.yaml",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-09-21T16:18:09.853552Z",
            "object_refs": [
                "identity--acf55024-6bbe-486f-a27a-7967559324f4",
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9",
                "indicator--6c0df49d-02c8-40be-aafe-fc0d1ab67eee",
                "indicator--5f1453be-7a5c-489a-8a5d-b3cb237a67ba",
                "relationship--ecce5385-4873-4d35-a8d5-50224341327c",
                "relationship--3d98bd94-546e-4cc2-8e25-8e40daaa3c57",
                "extension-definition--c8ea5ecb-f4a3-45e7-94de-9b9ba05161af"
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        }
    ]
}
```
